client/src/pages/Details.js

```diff
@@ -0,0 +1,172 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuid4 } from 'uuid';
+import { isValidEntry } from '../lib/validateFunctions';
+
+import EntryCard from '../components/EntryCard';
+import { UnfoldIcon } from '../icons/UnfoldIcon';
+
+export default function Details({ member, updateMember, members }) {
```

Comment:
Is this a component which shows details or a form? I thought that it would only show details of a member. Maybe the name is a bit misleading.

Reply:
Well, actually both. The page shows the details of a member but you can also add some notes via a form..